src/Solr/Stores/SolrConfigStore_File.php

```diff
@@ -49,4 +62,4 @@ public function instanceDir($index)
     {
         return $this->remote . '/' . $index;
     }
-}
+}
```

Comment:
Please keep the new line at the end of the file, thanks

Reply:
done